src/hooks/useEvent.ts

```
@@ -0,0 +1,38 @@
+import { eventUpdater } from "../apis/eventUpdater";
+import { fetcher } from "../apis/fetcher";
+import type {
+  EventApplyDtoType,
+  EventDtoType,
+  ParticipantValidationDto,
+  ParticipantValidationResponse,
+} from "../types/event";
+
+export const useEvent = () => {
+  const eventFetcher = (eventId: string | undefined) => {
+    const eventFetcher = useSWR<EventDtoType, AxiosError>(
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)